site/plugins.json

```diff
@@ -476,11 +485,15 @@
       },
       {
         "version": "3.9.2",
-        "siteDependencies": { "next": "<10.0.9" }
+        "siteDependencies": {
```

Comment:
This is due to me using a GitHub action to submit the plugin https://github.com/netlify/submit-build-plugin-action